SECURITY.md

```diff
@@ -0,0 +1,21 @@
+# Security Policy
+
+## Supported Versions
+
+Use this section to tell people about which versions of your project are
+currently being supported with security updates.
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 5.1.x   | :white_check_mark: |
+| 5.0.x   | :x:                |
+| 4.0.x   | :white_check_mark: |
+| < 4.0   | :x:                |
```

Comment:
Maybe this doesn't belong to "Security" problem but something like "Maintance Timetable"?

Reply:
All of this information came off of a family members router page it was injected into the page and  fonts with a bad certificate ..the whole page.
How can I get the vulnerabilities off of there?..

Reply:
> All of this information came off of a family members router page it was injected into the page and fonts with a bad certificate ..the whole page. How can I get the vulnerabilities off of there?..

You can directly leave your info by sending a message directly to [fengmk2@gmail.com](mailto:fengmk2@gmail.com), he is the leader of the project, if u confirm this has something to do with the vulnerabilities, plz attach the img if possible.

Reply:
> Maybe this doesn't belong to "Security" problem but something like "Maintance Timetable"?

Is there anything I can do to get this off my router login page and get it to stop touting through all my apps?

Reply:
> > All of this information came off of a family members router page it was injected into the page and fonts with a bad certificate ..the whole page. How can I get the vulnerabilities off of there?..
> 
> 
> 
> You can directly leave your info by sending a message directly to [fengmk2@gmail.com](mailto:fengmk2@gmail.com), he is the leader of the project, if u confirm this has something to do with the vulnerabilities, plz attach the img if possible.

I have had it in there a long time I was trying to upload files so people would know what was put on there because of course every technician at Hughes net will say it's not their fault or they don't see a problem.. it looks like someone maybe be just spying/ routing my data